src/middleware/withAuthorizedSpace.js

```diff
@@ -46,7 +49,7 @@ export function withAuthorizedSpace (handler) {
       ctx.authToken === null
 
     if (shouldServeLegacy) {
-      return handler(request, env, { ...ctx, space: null })
+      return handler(request, env, { ...ctx, space: undefined })
```

Comment:
Not really. We were explicitly setting it to null | undefined because it’s a legacy request (one without associated delegations). I believe it’s safe to remove that now.